db/osqlcomm.c

```diff
@@ -5111,7 +5105,7 @@ int osql_comm_signal_sqlthr_rc(osql_sess_t *sorese, struct errstat *xerr,
         max = OSQLCOMM_DONE_XERR_RPL_LEN;
     if (OSQLCOMM_DONE_RPL_LEN > max)
         max = OSQLCOMM_DONE_RPL_LEN;
-    buf = alloca(max);
```

Comment:
I know not introduced in this PR and noted in prior code-comment about being kludgy - runtime decision for buf size. Perhaps not even a performance issue. However, while we are here can use a union to clean this up:
```
union {
   char a[OSQLCOMM_DONE_XERR_UUID_RPL_LEN];
   char b[OSQLCOMM_DONE_UUID_RPL_LEN];
   char c[OSQLCOMM_DONE_XERR_RPL_LEN];
   char d[OSQLCOMM_DONE_RPL_LEN];
} u;
uint8_t *buf = (uint8_t *) &u;
```

Reply:
I like this idea.